pandas/core/base.py

```diff
@@ -877,6 +877,10 @@ def _values(self):
         """ the internal implementation """
         return self.values
 
+    @property
+    def empty(self):
+        return not bool(self.size)
```

Comment:
yeah, I didn't mean to imply that the NDFrame impl worked here, I meant that we could push the NDFrame impl into a subclass (e.g. into an intermediate class between `NDFrame` and `DataFrame`/`Panel`).  I doubt that's worth the added complexity though.